### DISCUSSION TITLE
Increasing Number of Bugs in Vuetify 3 ??

### DISCUSSION THREAD

**Discussion by KareemDa** [vuetifyjs/vuetify#16883] [General]
Hi Vuetify team,

I have been using the Vuetify library for four years now, and I am also a contributor to it. I have noticed that with each new release, more and more issues are appearing. I wanted to raise this issue with the team in the hopes that we can find a solution.

While I appreciate the hard work and effort that goes into developing and maintaining Vuetify, I believe that the recent releases have been too buggy. These bugs are causing problems for me and other users, and it's affecting our productivity.

I wanted to start this discussion to see if others are experiencing the same issues and to find out if there is a plan to address them. I know that the Vuetify team is committed to delivering a high-quality library, and I hope that we can work together to resolve these issues.

Thank you for your attention, and I wish Vuetify all the best in its future development.

Best regards,

**Comment by mohammad-zr** [comment]
We have same problem too. thanks too vuetify developers for their hard work but I think they should not announce Vuetify 3 officially. It's very buggy and also it's not clear when full stable version get released...
